pyproject.toml

```diff
@@ -29,11 +29,11 @@ classifiers = [
     "Topic :: Security",
 ]
 dependencies = [
-    "pydantic~=2.9.2",
-    "python-dotenv~=1.0.1",
-    "requests~=2.32.3",
-    "rich~=13.7.1",
-    "shodan~=1.31.0",
+    "pydantic<3",
+    "python-dotenv<2",
+    "requests<3",
+    "rich<14",
+    "shodan<2",
```

Comment:
Thanks for the PR! I like this, but maybe it's still good to be explicit with the minimum version number. For example:

```
    "pydantic>=2.9.2,<3",
```